.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: CI testing
+on:
+  pull_request: {}
+  push:
+    branches:
+      - main
+      - release-candidate-*
+    tags:
+      - v*
+  workflow_dispatch:
+    inputs:
+      debug_enabled:
+        description: Run the build with SSH debugging enabled
+        required: false
+        default: false
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    strategy:
+      matrix:
+        include:
+          - script: static
+          - script: test
+            coverage-name: test
+          - script: docs
+      # TODO: get this working again
```

Comment:
So this is using the local docs build instead of the pervious approach of running through slurm. Do you know what difference that makes?

Reply:
I didn't look into it in any detail really, I just wanted CI to be green, and there wasn't anything I could SSH into that would work. I'm pretty sure that there are non-FPGA implementations of everything in NengoFPGA, so it should still be functionally similar, but @xchoo would know better.

Reply:
Messaged Xuan, he said building the docs without access to a physical FPGA is fine.